apps/gp2-frontend/src/users/__tests__/export.test.ts

```diff
@@ -0,0 +1,233 @@
+      ...gp2Fixtures.createUserResponse(),
+      country: 'Portugal',
+      city: 'Lisbon',
+    };
+    const outputWithoutCity: gp2Model.UserResponse = {
+      ...gp2Fixtures.createUserResponse(),
+      country: 'Portugal',
+      city: undefined,
+    };
+    expect(userToCSV(outputWithCity).location).toMatchInlineSnapshot(
+      `"Portugal, Lisbon"`,
+    );
+    expect(userToCSV(outputWithoutCity).location).toMatchInlineSnapshot(
+      `"Portugal"`,
+    );
+  });
+  it('flattens and order degrees', () => {
+    const output: gp2Model.UserResponse = {
+      ...gp2Fixtures.createUserResponse(),
+      degrees: ['MBA', 'BA', 'PhD'],
+    };
+    expect(userToCSV(output).degrees).toMatchInlineSnapshot(`
+      "BA,
+      MBA,
+      PhD"
+    `);
+  });
+  it.each<{ value: boolean; valueOutput: string }>([
+    { value: true, valueOutput: 'Yes' },
+    { value: false, valueOutput: 'No' },
+  ])(
+    'transforms the boolean $value into $valueOutput',
+    ({ value, valueOutput }) => {
+      const output: gp2Model.UserResponse = {
+        ...gp2Fixtures.createUserResponse(),
+        onboarded: value,
+      };
+      expect(userToCSV(output).onboarded).toBe(valueOutput);
+    },
+  );
+  it('flattens the first three positions into primaryPosition, secondaryPosition and terciaryPosition', () => {
+    const output: gp2Model.UserResponse = {
+      ...gp2Fixtures.createUserResponse(),
+      positions: [
+        {
+          role: 'CEO',
+          department: 'Research',
+          institution: 'Stark Industries',
+        },
+        {
+          role: 'CTO',
+          department: 'Technology',
+          institution: 'YLD',
+        },
+        {
+          role: 'CFO',
+          department: 'Finance',
+          institution: 'Bank of America',
+        },
+      ],
+    };
+    expect(userToCSV(output).primaryPosition).toMatchInlineSnapshot(
+      `"CEO in Research at Stark Industries"`,
+    );
+    expect(userToCSV(output).secondaryPosition).toMatchInlineSnapshot(
+      `"CTO in Technology at YLD"`,
+    );
+    expect(userToCSV(output).terciaryPosition).toMatchInlineSnapshot(
+      `"CFO in Finance at Bank of America"`,
+    );
+  });
+  it('positions are left empty if there are no positions', () => {
+    const output: gp2Model.UserResponse = {
+      ...gp2Fixtures.createUserResponse(),
+      positions: [],
+    };
+    expect(userToCSV(output).primaryPosition).toBeUndefined();
+    expect(userToCSV(output).secondaryPosition).toBeUndefined();
+    expect(userToCSV(output).terciaryPosition).toBeUndefined();
+  });
+  it('flattens and orders projects', () => {
+    const output: gp2Model.UserResponse = {
+      ...gp2Fixtures.createUserResponse(),
+      projects: [
+        { id: '1', title: 'project 1', status: 'Active' as const, members: [] },
+        { id: '2', title: 'project 2', status: 'Active' as const, members: [] },
+        { id: '3', title: 'project 3', status: 'Active' as const, members: [] },
```

Comment:
I don't think we need the as const

```suggestion
        { id: '1', title: 'project 1', status: 'Active', members: [] },
        { id: '2', title: 'project 2', status: 'Active', members: [] },
        { id: '3', title: 'project 3', status: 'Active', members: [] },
```